yt_dlp/extractor/hotstar.py

```diff
@@ -332,6 +333,7 @@ def _real_extract(self, url):
             'season_id': video_data.get('seasonId'),
             'episode': video_data.get('title'),
             'episode_number': int_or_none(video_data.get('episodeNo')),
+            'thumbnails': video_data.get('imageSets'),
```

Comment:
Transform the data to correct field names we want using `traverse_obj`. See docstring of InfoExtractor in common.py